Extensions/EdFi.Ods.Extensions.SampleStudentTranscript/EdFi.Ods.Extensions.SampleStudentTranscript.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Same comment here about updating `Autofac.Extensions.DependencyInjection` to `7.2.0` and `Newtonsoft.Json` to `13.0.1`

Reply:
updated